### PR TITLE
Test/223 create test for habit assign controller

### PR DESCRIPTION
**Pull request by Andrii-Danylenko**
Created test class for habit assign controller

### CODE REVIEW

**Review by SezamUA — COMMENTED** [review]
Maybe it's a good idea to use in verify(...) expected values according to request in mockMvc.perform(..). What if there is some error in controller's method and it calls service with wrong parameters.

**Review by StannyB9 — APPROVED** [review]

**Review by DZotsenko — APPROVED** [review]

**Review by DenisOgr — APPROVED** [review]

File: core/src/test/java/greencity/controller/HabitAssignControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(MockitoExtension.class)
+public class HabitAssignControllerTest {
+    @Mock
+    UserService userService;
```
Comment (maks741) [inline core/src/test/java/greencity/controller/HabitAssignControllerTest.java:37]:
```suggestion
    private UserService userService;
```

File: core/src/test/java/greencity/controller/HabitAssignControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+    @Mock
+    HabitAssignServiceImpl habitAssignService;
```
Comment (maks741) [inline core/src/test/java/greencity/controller/HabitAssignControllerTest.java:39]:
```suggestion
    private HabitAssignServiceImpl habitAssignService;
```

File: core/src/test/java/greencity/controller/HabitAssignControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+    @InjectMocks
+    HabitAssignController habitAssignController;
```
Comment (maks741) [inline core/src/test/java/greencity/controller/HabitAssignControllerTest.java:41]:
```suggestion
    private HabitAssignController habitAssignController;
```

File: core/src/test/java/greencity/controller/HabitAssignControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+    @Mock
+    ModelMapper modelMapper;
```
Comment (maks741) [inline core/src/test/java/greencity/controller/HabitAssignControllerTest.java:43]:
```suggestion
    private ModelMapper modelMapper;
```

**Review by maks741 — APPROVED** [review]

**Pull request closed without merging**